.taskcluster.yml

```diff
@@ -85,7 +85,7 @@ tasks:
                   git clone ${repo_url} balrog &&
                   cd balrog &&
                   git checkout ${head_sha} &&
-                  pip install coveralls==1.9.2 tox &&
+                  pip install coveralls tox &&
```

Comment:
Shall we move coveralls to `test.in`? It doesn't belong there logically (you don't necessarily submit to coveralls as a   part of every test run), but at least it's pinned.

Reply:
Yeah, that's probably a good idea. I'll try to do it here.